.circleci/config.yml

```diff
@@ -70,6 +75,17 @@ jobs:
       - run: go test -v -race ./... -coverprofile=coverage.txt -covermode=atomic
       - codecov/upload:
           file: coverage.txt
+  test_1_14:
+    working_directory: /go/src/github.com/nspcc-dev/neo-go
+    executor: go1_14
+    steps:
+      - checkout
+      - run: git submodule sync
+      - run: git submodule update --init
+      - gomod
+      - run: go test -v -race ./... -coverprofile=coverage.txt -covermode=atomic
+      - codecov/upload:
+          file: coverage.txt
```

Comment:
I guess only one test run should submit coverage to Codecov, not two. And I think it's more appropriate to upload numbers from the latest go.